types/underscore/index.d.ts

```diff
@@ -4881,6 +4909,12 @@ declare namespace _ {
          */
         compose(...functions: Function[]): Function;
 
+        /**
+         * Wrapped type `Function`.
+         * @see _.restArguments
+         */
+        restArguments(func:Function, startIndex:number): Function;
```

Comment:
Could this type not be more precise? `_.restArguments` modifies the parameter list of a function, and I think TS might be able to express this.

Also, why did you add this to the `UnderscoreStatic` interface but not to the other two?

Reply:
I will report an error when using it on my end, and it has been corrected so far